presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java

```diff
@@ -575,6 +575,15 @@ public void testUnnest()
         assertQuery("SELECT b FROM UNNEST(ARRAY[1, 2, 3], ARRAY[4, 5]) t(a, b)", "SELECT * FROM VALUES 4, 5, NULL");
         assertQuery("SELECT count(*) FROM UNNEST(ARRAY[1, 2, 3], ARRAY[4, 5])", "SELECT 3");
         assertQuery("SELECT a FROM UNNEST(ARRAY['kittens', 'puppies']) t(a)", "SELECT * FROM VALUES ('kittens'), ('puppies')");
+        assertQuery(
+                "SELECT * FROM UNNEST(ARRAY[ROW(1, 2), ROW(3, 4)], ARRAY[ROW('a', true), ROW('b', false)])",
```

Comment:
in case of 
```
SELECT * FROM UNNEST(ARRAY[ROW(1, 2), ROW(3, 4)])
```
what are the column names?

Reply:
They shouldn’t have one, just like every other place in the engine that produces anonymous rows or columns.